### DISCUSSION TITLE
Reservoir range adjustable

### DISCUSSION THREAD

**Discussion by githubfan1** [Keriew/augustus#717] [General]
I modified the reservoir range to 16 tiles and I am pleased with the change. More space available, more compact layout, no more undesirable reservoirs around, less aqueducts, anyways, I just liked it. It should be a configurable item, 10 tiles (default) to, say, 20 tiles.

**Comment by githubfan1** [comment]
![reservoir-config-range](https://user-images.githubusercontent.com/6599012/195998489-92db6ad2-97d0-492c-8e10-16ece7aef1e8.png)

**Comment by Keriew** [comment]
Hey, thanks for the feedback!

I think the Reservoir range is in a good spot right now. We also have a Neptune Grand Temple which increases the reservoir range, if players really find that constricting on some maps. While reservoir and aqueducts usually aren't too important or challenging aspect of the game, I feel they are justified and important from the flavor point of view - aqueducts being a very iconic part of Roman engineering.

That being said I think that it'd be nice at some point to export many of the hardcoded game constants to be available to change for the player, if they wanted to mess around with the game on their own. 

(Also, be sure to adjust RESERVOIR_RANGE_MAX_TILES in ```src/widget/city_building_ghost``` when increasing the reservoir range, I remember we had to deal with an insidious memory corruption glitch the last time we adjusted it.)
